distribution/resources/conf/micro-gw.conf

```diff
@@ -43,6 +43,9 @@ task.uploadFiles=true
 username="admin"
 password="admin"
 
+[http2]
+http2Enable=false
```

Comment:
Since this is under http2 tag I think we do not need to mention it again as "http2Enable". We can simply say "Enable"